blacklite-core/src/main/java/com/tersesystems/blacklite/AsyncEntryWriter.java

```diff
@@ -139,7 +188,7 @@ public void close() throws Exception {
         });
 
     // Give the executor a second to commit and close out cleanly...
-    if (!executor.awaitTermination(1L, TimeUnit.SECONDS)) {
+    if (!executor.awaitTermination(1000L, TimeUnit.SECONDS)) {
```

Comment:
For executors that have REALLY LARGE queues to archive and write, 1 second isn't enough.